cmd/state-installer/installer.go

```diff
@@ -116,6 +116,11 @@ func run(out output.Outputer, installPath, sessionToken string, updateTag *strin
 		}
 	}
 
+	err = cfg.Set(installation.ConfigKeyInstallPath, installPath)
+	if err != nil {
+		logging.Error("Could not set install path in config: %v", errs.JoinMessage(err))
+	}
```

Comment:
This isn't necessary. By definition people hitting our use-case will use the default install path.